core/templates/services/context.service.spec.ts

```diff
@@ -160,6 +169,11 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
```

Comment:
This test may not catch regressions properly. `canEntityReferToSkills` returns true if the page context is one of 3 pages. Can we split this test into separate tests to ensure that it returns true only for the 3 pages mentioned and not for the others. Please add necessary prechecks and post checks in these tests.

Reply:
Note: there may also be a condition where the context is the editor page but the exploration is not linked to a story.

Reply:
Done!

Reply:
Should we change the description to be more specific now that we have multiple tests that will assert `expect(ecs.canEntityReferToSkills()).toBe(true);`?

Reply:
Done!

Reply:
Not done. I don't see that the description has been edited.

Reply:
Thanks for pointing out. Done!